maintainers/README.md

```diff
@@ -172,3 +172,49 @@ to the team without an approval by at least one existing member.
 Various utility scripts, which are mainly useful for nixpkgs maintainers,
 are available under `./scripts/`.  See its [README](./scripts/README.md)
 for further information.
+
+# Guidelines for Committers
+
+When merging pull requests, care must be taken to reduce impact to the `master`
+branch. If a commit breaks evaluation, it will affect Ofborg evaluation results
+in other pull requests and block Hydra CI, thus introducing chaos to our
+workflow.
+
+One approach to avoid merging such problematic changes is to wait for
+successful Ofborg evaluation. Additionally, using tools like `nixpkgs-review`
+can help spot issues early, before Ofborg finishes evaluation.
+
+## Breaking changes
+
+In general breaking changes to `master` and `staging` branches are permitted.
+Though restrictions might apply towards the end of a NixOS release cycle, due
+to our feature freeze mechanism. This is to avoid large-scale breakages shortly
+before and during a Zero Hydra Failures (ZHF) campaign. These restrictions also
```

Comment:
should we explain what "ZHF" is, or is that too out of scope? Maybe instead we could link to somewhere that mentions ZHF?

Reply:
I think it's usually explained in the release schedule/ZHF issue and as that is a moving target I think it's okay to not link to anything here